src/content/app/tools/blast/components/blast-species-selector/BlastSpeciesSelector.tsx

```diff
@@ -46,6 +53,10 @@ const BlastSpeciesSelector = () => {
     }
   };
 
+  const availableSpeciesList = popularSpeciesList.filter(
+    (species) => species.scientific_name
```

Comment:
This is the field to use:

![image](https://user-images.githubusercontent.com/6834224/199771364-8548051a-d3e8-4699-ac90-93276c97e708.png)

Reply:
Oh cool. Thanks!